src/main/java/net/sf/jabref/logic/importer/fetcher/GoogleScholar.java

```diff
@@ -62,4 +88,81 @@
 
         return pdfLink;
     }
+
+    @Override
+    public String getName() {
+        return "Google Scholar";
+    }
+
+    @Override
+    public HelpFile getHelpPage() {
+        return HelpFile.FETCHER_GOOGLE_SCHOLAR;
+    }
+
+    @Override
+    public List<BibEntry> performSearch(String query) throws FetcherException {
+        try {
+            obtainAndModifyCookie();
+
+            List<BibEntry> foundEntries = new ArrayList<>(10);
+
+            String queryURL = String.format(BASIC_SEARCH_URL, URLEncoder.encode(query, StandardCharsets.UTF_8.name()));
+            addHitsFromQuery(foundEntries, queryURL);
+            String content;
+
+            if(foundEntries.size()==10) {
+                String secondPage = queryURL+"&start=10";
+                addHitsFromQuery(foundEntries, secondPage);
+            }
+
+            return foundEntries;
+        } catch (IOException e) {
+            throw new FetcherException("Fetching entries from Google Scholar failed: ", e);
```

Comment:
The error message should probably localized (applies also to the code below). I think there is similar error handling in SearchBasedParserFetcher.